test/unit/jest.config.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
 		'<rootDir>/.*/build/',
 		'<rootDir>/.*/build-module/',
 		'<rootDir>/.+.native.js$',
+		'packages/react-native-*',
```

Comment:
Really minor but we could try to be a bit more consistent with `/packages/react-native-*`

Reply:
Updated 👍